Header header

float32 localization_error_em_

float32 localization_error_dr_

float32 mapping_error_em_

float32 mapping_error_dr_
